src/popup/components/TokenAmount.vue

```diff
@@ -59,23 +60,30 @@ export default {
     noSymbol: Boolean,
     highPrecision: Boolean,
   },
-  computed: {
-    amountRounded() {
-      if (Number.isInteger(this.amount)) return this.amount;
-      if (this.amount === 0) return this.amount;
-      return this.amount.toFixed((this.highPrecision || this.amount < 0.01) ? 9 : 2);
-    },
-    ...mapState({
-      amountFiat(state, { convertToCurrency, formatCurrency }) {
-        if (this.hideFiat || this.aex9) return '';
-        const converted = convertToCurrency(this.amount);
-        if (this.amount === 0) return formatCurrency(0);
-        if (converted < 0.01) return `<${formatCurrency(0.01)}`;
-        return `≈${formatCurrency(converted)}`;
-      },
-    }),
+  setup(props) {
+    const { convertToCurrency, formatCurrency } = useCurrencies();
+
+    const amountRounded = computed(() => {
+      if (Number.isInteger(props.amount) || props.amount === 0) {
+        return props.amount;
+      }
+      return props.amount.toFixed((props.highPrecision || props.amount < 0.01) ? 9 : 2);
+    });
+
+    const amountFiat = computed((): string => {
+      if (props.hideFiat || props.aex9) return '';
+      const converted = convertToCurrency(props.amount);
+      if (props.amount === 0) return formatCurrency(0);
+      if (converted < 0.01) return `<${formatCurrency(0.01)}`;
+      return `≈${formatCurrency(converted)}`;
+    });
```

Comment:
Please wrap the returns with braces

Reply:
Sure. I also see that this piece of code already exists. So I'll reuse it as an method.